kubernetes-client-openapi-discovery/src/main/java/io/micronaut/kubernetes/client/openapi/KubernetesConfiguration.java

```diff
@@ -307,6 +310,20 @@ public boolean isWatch() {
         public void setWatch(boolean watch) {
             this.watch = watch;
         }
+
+        /**
+         * @return whether to terminate service startup if not able to load config from kubernetes
```

Comment:
specify in the javadoc what the default is

Reply:
done